Documentation/contributor-guide/release.md

```diff
@@ -73,3 +73,12 @@ git log ...${PREV_VERSION} --pretty=format:"%an" | sort | uniq | tr '\n' ',' | s
 
 - Create new stable branch through `git push origin ${VERSION_MAJOR}.${VERSION_MINOR}` if this is a major stable release. This assumes `origin` corresponds to "https://github.com/etcd-io/etcd".
 - Bump [hardcoded Version in the repository](https://github.com/etcd-io/etcd/blob/v3.4.15/version/version.go#L30) to the version `${VERSION}+git`.
+
+## Patch release criterion
+
+Consider releasing a patch version if any of the following conditions stand:
```

Comment:
Suggest strengthening the language a bit.

```suggestion
The etcd project aims to release a new patch version if any of the following conditions are met:
```

Reply:
Done.